cmd/earth/main.go

```diff
@@ -1201,8 +1201,13 @@ func (app *earthApp) actionBuild(c *cli.Context) error {
 	}
 	secretsMap[debuggercommon.DebuggerSettingsSecretsKey] = debuggerSettingsData
 
+	sc, err := secretsclient.NewClient()
+	if err != nil {
+		app.console.Warnf("failed to create secrets client: %v\n", err)
```

Comment:
this will occur in CI environments, or any environments without ssh-agent running.

Reply:
Oh yes.. that's a good point. I'm thinking we might have to support token-based auth too sooner because of CI environments.